tests/pipelines/fetchdatairidanext.nf.test

```diff
@@ -54,4 +54,58 @@ nextflow_pipeline {
             assert path("$launchDir/results/reads/SRR999934_2.fastq.gz").exists() == false
         }
     }
+
+    test("integration test with ENA") {
+
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheet.csv"
+                outdir = "test3_out"
+                provider = "ENA"
+            }
+        }
+
+        then {
+            assert workflow.success
+
+            // IRIDA Next output file
+            assert path("$launchDir/test3_out/iridanext.output.json").json == path("$baseDir/tests/data/test1_iridanext.output.json").json
+
+            // Output data
+            assert path("$launchDir/test3_out/reads/ERR1109373_1.fastq.gz").linesGzip.size() == 512
+            assert path("$launchDir/test3_out/reads/ERR1109373_2.fastq.gz").linesGzip.size() == 512
+            assert path("$launchDir/test3_out/reads/SRR13191702_1.fastq.gz").linesGzip.size() == 364
+            assert path("$launchDir/test3_out/reads/SRR13191702_2.fastq.gz").linesGzip.size() == 364
```

Comment:
I know this is a pain, but could you peek into these files and ensure that the first FASTQ record ID matches what's expected, at least maybe a partial ID or something like that in there?

Reply:
Could probably add `md5` checksum instead if we wanted to check the file contents, which you can do like https://www.nf-test.com/docs/assertions/files/#md5-checksum

Reply:
I implemented the md5 checksum instead of the line count